### PR TITLE
Adding content for BART Migration Journey Document

### PR DESCRIPTION
**Pull request by abhilasha-narendra**
## What Changed?

Adding content for the document.

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [x] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content

### CODE REVIEW

**Comment by drothery-edb** [comment]
@josh-heyer this is ready to be merged. Adam has reviewed it and Kelly approved publishing it. I just did a rebase. Do you want to do a quick review?

File: product_docs/docs/bart/2.6/bart_migration/06_scheduling.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    | Barman     | `barman backup example --reuse-backup=link`<br/>This can also be set in the configuration file at the global/server level with `reuse_backup = link`|
+    | pgBackRest | Incremental:`pgbackrest backup --type=incr --stanza=example`<br/>Differential: `pgbackrest backup --type=diff --stanza=example`|
+
+
+### Retention management
+
+-   With BART usually a cron job is set up to run maintenance on the
+    > BART repository (bart MANAGE). By supplying the -d option, the
+    > obsoleted backups are automatically cleaned (together with their
+    > WAL archives).
```
Comment (josh-heyer) [inline product_docs/docs/bart/2.6/bart_migration/06_scheduling.mdx:44]: I don't *think* this is intentional (blockquoting the bulk of each paragraph); I'd assume the intent here was to group each paragraph as a list item; you need only indent for that to happen.

```suggestion
   BART repository (bart MANAGE). By supplying the -d option, the
   obsoleted backups are automatically cleaned (together with their
   WAL archives).
```

File: product_docs/docs/bart/2.6/bart_migration/06_scheduling.mdx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+-   Similarly, Barman uses a cron command to run maintenance on the
+    > Barman repository \`barman cron\`. The \`barman cron\` command
+    > takes care of more things (like copying streamed WAL files to the
+    > WAL archive directory), and needs to be scheduled to run every
+    > minute. The Barman rpm or debian package automatically creates a
+    > cron entry running every minute as the barman user.
```
Comment (josh-heyer) [inline product_docs/docs/bart/2.6/bart_migration/06_scheduling.mdx:51]: (see previous comment re: blockquotes & lists)

```suggestion
   Barman repository \`barman cron\`. The \`barman cron\` command
   takes care of more things (like copying streamed WAL files to the
   WAL archive directory), and needs to be scheduled to run every
   minute. The Barman rpm or debian package automatically creates a
   cron entry running every minute as the barman user.
```

File: product_docs/docs/bart/2.6/bart_migration/06_scheduling.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+-   pgBackRest runs maintenance with the expire command (pgbackrest
+    > expire), but the expire command is run automatically after each
+    > successful backup, and is not required to be separately scheduled.
```
Comment (josh-heyer) [inline product_docs/docs/bart/2.6/bart_migration/06_scheduling.mdx:55]: (see previous comment re: blockquotes & lists)
```suggestion
   expire), but the expire command is run automatically after each
   successful backup, and is not required to be separately scheduled.
```

**Review by josh-heyer — CHANGES_REQUESTED** [review]

**Review by josh-heyer — APPROVED** [review]
Fixed minor issue (see resolved comments) - should be good to go now!

**Pull request merged**